draft-ietf-jsonpath-base.md

```diff
@@ -929,24 +857,24 @@ This section is non-normative.
 
 Array indexing is a way of selecting a particular element of an array using
```

Comment:
Can we say this is the same as with an index selector?

Reply:
I think is simpler fix is to delete the index selector stuff from this section. See 977965ad795158e802bfbf82c0f1a62c7b0b9a2f.

Reply:
Thanks.  I note that there is still some repetition in the semantics section, but we can fix that later.